src/Shared/MSBuild.Testing/ProjectDirectory.cs

```diff
@@ -37,7 +37,7 @@ public ProjectDirectoryOptions(string baseDirectory, string targetProjectName, s
 
         public static ProjectDirectory Create(string originalProjectName, ProjectDirectoryOptions options, params string[] additionalProjects)
```

Comment:
Probably this directory too. I don't think we have any msbuild tests left in this repo